modules/collection/programs/helix.nix

```
@@ -90,11 +90,11 @@ in {
     packages = mkIf (cfg.package != null) [cfg.package];
     files =
```

```suggestion
    xdg.config.files =
```

Did this not cause the test to fail?